### DISCUSSION TITLE
umi-ocr有否考虑换个图标？基于原来的淡粉和低调绿色构成，简练，延伸变化...

### DISCUSSION THREAD

**Discussion by lforlgg** [hiroi-sora/Umi-OCR#484] [Ideas]
umi-ocr有否考虑换个图标？基于原来的淡粉和低调绿色构成，简练，延伸变化...💖

💛结构简单明了，更有质感，现代感一些。
💛U字形象为主体。
💛Text、Ocr的大写为主构成，图形文化的含义在内。
💛T臂膀粗壮，寓意解构稳健。

我随便弄弄，非专业人士。
umi-oc的图标，需要一笔画龙点睛，抛砖引玉，希望引起相关专业热心人士的贡献。🤭

![4Umi-Ocr](https://github.com/hiroi-sora/Umi-OCR/assets/105794681/338e60ac-67d8-4251-90ca-43359f6fd44c)

![Windesktop](https://github.com/hiroi-sora/Umi-OCR/assets/105794681/574c865a-95ac-47d1-abda-da45dcfe998d)

[Free4UmiOcr.zip](https://github.com/hiroi-sora/Umi-OCR/files/15051350/Free4UmiOcr.zip)

**Comment by hiroi-sora** [comment]
其实，Umi的名称和图标，有一个含义是致敬 [Dango-Translator](https://github.com/PantsuDango/Dango-Translator) 这个项目。

换图标的话，或许以后才会考虑吧……

**Comment by lforlgg** [comment]
👍重情重义，原来有更深的含义。
冒昧了。😊
我以为我这贴没发成功，原来在讨论区里了。
